src/lucky_migrator/primary_key_type.cr

```diff
@@ -2,5 +2,13 @@ module LuckyMigrator
   enum PrimaryKeyType
     Serial
     UUID
+
+    def db_type
```

Comment:
Not very proud on that method. Can we do sth more elegant here? Some type of mapping table?